app/src/main/java/org/y20k/transistor/core/Station.java

```diff
@@ -432,6 +460,10 @@ else if (line.startsWith("Title1=")) {
                 mStreamUri = Uri.parse(line.substring(6).trim());
             }
 
+            // Use the first entry, some stations shuffle playlist to put less used mirror to the top of the list
```

Comment:
Previously it used the last found HTTP URL, which has even less guarantee to be streamable, because most radio stations put their good mirror at the top of .m3u list.

This particular station has their .mp3 stream as a first entry, and .aac stream as a last entry, which we cannot play.

Ideally we should implement a selection dialog when the playlist contains multiple entries.

I have no comments about malformed .m3u.